### PR TITLE
New developer boards section and description for the HiHope SCDAY200 board

### PR DESCRIPTION
**Pull request by Stefan-Schmidt**
New section to hold dev board information. DAY200 description, specs and flashing description. Moving parts form the quick build to proper places.

### CODE REVIEW

File: dev-boards.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: Developer Boards
+nav_order: 2
+has_children: true
+---
+
+# Developer Boards supported by Eclipse Oniro
+
+This section goes into the details of the Eclipse Oniro supported developer
+boards. Supported here means in active use by its developers and parts of the
+CI/CD pipeline guarding the development.
```
Comment (frankplus) [inline dev-boards.md:12]:
```suggestion
This section delves into the specifics of Eclipse Oniro's supported developer 
boards. 'Supported' here means being in active use by its developers and being part of the 
CI/CD pipeline that guards the development processes.
```

File: dev-boards.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Currently this only cover the [HiHope SCDAY200 development board](hh-scdayu200.html).
```
Comment (frankplus) [inline dev-boards.md:14]:
```suggestion
Currently this only cover the [HiHope SCDAYU200 development board](hh-scdayu200.html).
```

File: hh-scdayu200.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+layout: default
+---
+
+# HiHope HH-SCDAYU200 Development Kit
+
+## Introduction
+Based on the Rockchip RK3568, this development kit integrates advanced features
+such as a dual-core architecture GPU and a high-performance NPU. Combined with
+the onboard quad-core 64-bit Cortex-A55 processor which has a frequency of up
+to 2.0GHz.
```
Comment (frankplus) [inline hh-scdayu200.md:14]: Lack of verb:
```suggestion
such as a dual-core architecture GPU and a high-performance NPU. Complemented by 
an onboard quad-core 64-bit Cortex-A55 processor which has a frequency of up to 2.0GHz, 
it ensures robust performance.
```

File: hh-scdayu200.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+| **LEDs**          | 3x LED |
+| **Sensors**       | Bosch Sensortec BMA456, BMI270 and BMP581 |
+| **Fan**           | 1x Fan |
+
+## Building
+
+To build Eclipse Oniro for this board the normal [quick build](quick-build.html)
+procedure has to be used to fetch the needed source code and environment.
+
+At the actual build step, inside the Docker instance, the target device for the
```
Comment (frankplus) [inline hh-scdayu200.md:76]:
```suggestion
During the build step, inside the Docker instance, the target device for the
```

File: hh-scdayu200.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+After this setup, running `flash/flash.py -q` should produce the following output, indicating readiness:
+
+```bash
+maskrom
+```
+
+To enable *programming mode* on the device, perform the following steps:
+
+ 1. Press nad hold `VOL/RECOVERY` then `RESET` buttons.
```
Comment (frankplus) [inline hh-scdayu200.md:122]:
```suggestion
 1. Press and hold `VOL/RECOVERY` then `RESET` buttons.
```

**Review by frankplus — COMMENTED** [review]
I left some suggested changes mainly typos and grammatical errors

**Comment by Stefan-Schmidt** [comment]
Thanks a lot for these changes @frankplus . I folded them into the original commits.

**Review by frankplus — APPROVED** [review]
LGTM

**Pull request merged**